index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <title>Apple</title>
+        <link rel="stylesheet" href="css/style.css" type="text/css" />
+        <script src="https://kit.fontawesome.com/1b0418f0c5.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <main class="main">
+            <nav class="nav-container">
+                <div class="first"><img class="apple" src="images-apple/logo-apple.png" alt="apple" /></div>
+                <div>
+                    <a href="#">
+                        <p>Store</p>
+                    </a>
```

Comment:
you have two options either you make the div as an <a tag or you can make the <a tag take all the width and height, by giving them 100% and use a flexbox inside to align the text inside, using justify-content and align-items to center.

### this applies for the whole navbar